systemtap/iotrace.stp

```diff
@@ -0,0 +1,39 @@
+#! /usr/bin/env stap
+# Usage: sudo stap iotrace.stp -x <pid>
+
+probe begin
+{
+	printf("begin to trace thread IO");
+}
+
+probe vfs.read
+{
+	if (target() == pid()) {
+		printf("%s[%ld] - read %d from %s\n", execname(), tid(), bytes_to_read, reverse_path_walk($file->f_path->dentry))
```

Comment:
seem bytes_to_read is not the real data read from IO, maybe we should use vfs.read.return to get the return size?